contracts/modules/andromeda-shunting/src/contract.rs

```diff
@@ -156,9 +156,9 @@ fn parse_params(deps: Deps, params: Vec<EvaluateParam>) -> Result<Vec<String>, C
                 }
                 .into();
 
-                let raw_result: Value = deps.querier.query::<Value>(&query_msg).unwrap();
-                let json = JSON::from(raw_result);
-                let Value::String(val) = json.get(&accessor).unwrap() else {
+                let raw_result: Value = deps.querier.query::<Value>(&query_msg)?;
+                let json = JSON::try_from(raw_result).unwrap();
```

Comment:
Is it possible to avoid using `unwrap`?

Reply:
Tried to avoid `unwrap` but it did not work. `JSON::from` was a bit tricky based on the rust version. On some versions it was accepting `Vaule` while on other versions, it was only accepting `JSON`. I think it might be better to use `try_from` and `unwrap` at the moment